src/components/UI/side-drawer/side-drawer.module.scss

```diff
@@ -16,7 +16,13 @@
   text-align: center;
   padding-top: 20px;
   color: #1d1283;
+  display: flex;
+  flex-direction: row-reverse;
 }
+.logo {
+  margin-right: 2.8125rem;
```

Comment:
Why are we using the value till this many decimal places ??
Can we please use pixel here ??

Reply:
Deleted sidebar component